python/dataproc_templates/jdbc/README.md

```diff
@@ -0,0 +1,112 @@
+# 1. JDBC To JDBC
+
+Template for reading data from JDBC table and writing them to a JDBC table. It supports reading partition tabels and write into partitioned or non-partitioned tables.
+
+## Arguments
+
+* `jdbctojdbc.input.url`: JDBC input URL
+* `jdbctojdbc.input.driver`: JDBC input driver name
+* `jdbctojdbc.input.table`: JDBC input table name
+* `jdbctojdbc.input.partitioncolumn`: JDBC input table partition column name
+* `jdbctojdbc.input.lowerbound`: JDBC input table partition column lower bound which is used to decide the partition stride
+* `jdbctojdbc.input.upperbound`: JDBC input table partition column upper bound which is used to decide the partition stride
+* `jdbctojdbc.numpartitions`: The maximum number of partitions that can be used for parallelism in table reading and writing. Default set to 10
```

Comment:
Also specify that same value will be used for both input and output jdbc connections.

Reply:
Updated the readme file.